src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -0,0 +1,41 @@
+(ns org.broadinstitute.firecloud-ui.common.style
+  (:require [org.broadinstitute.firecloud-ui.utils :as utils]))
+
+(def colors {:link-blue "#6690c5"
+             :line-gray "#e6e6e6"
+             :button-blue "#457fd2"
+             :background-gray "#f4f4f4"
+             :text-light "#7f7f7f"
+             :text-gray "#666"})
+
+
+(defn form-label [text]
```

Comment:
Is it too verbose to do `create-form-label`, `create-hint`, etc. to distinguish these from other things like `(def input-text-style ...)`, etc.?